getting-started/create-catalog.sh

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2024 Snowflake Computing Inc.
```

Comment:
Can we use Apache license instead? Same other new files. 

Relates to https://github.com/apache/polaris/pull/173

Reply:
Good catch!